lib/tackle/consumer/executor.ex

```diff
@@ -98,6 +98,11 @@ defmodule Tackle.Consumer.Executor do
     {:stop, {:connection_lost, reason}, state}
   end
 
+  # This message is received because of fetching OS certificates during connection opening in Tackle.Connection.open/1
+  def handle_info({:EXIT, _port, :normal}, state) do
```

Comment:
Pattern match for the Port would be nice, to handle _only_ the OS X Port EXIT messages.